core/src/main/java/io/undertow/server/handlers/resource/PathResource.java

```diff
@@ -57,6 +57,9 @@ public String getPath() {
     @Override
     public Date getLastModified() {
         try {
+            if (Files.isSymbolicLink(file) && Files.notExists(file)) {
+                return new Date(0L);
```

Comment:
Shouldn't this return null based on `getLastModified` docs? https://github.com/undertow-io/undertow/blob/ff4c9cf37872cb96070ba6a2fcbbaa6df291e390/core/src/main/java/io/undertow/server/handlers/resource/Resource.java#L47

Reply:
Agree, can you please update this @iweiss ?

Reply:
@fl4via How about now?